lib/kube/proxy/forwarder.go

```diff
@@ -2153,14 +2039,19 @@ func (f *Forwarder) getExecutor(ctx authContext, sess *clusterSession, req *http
 	return remotecommand.NewSPDYExecutorForTransports(rt, upgradeRoundTripper, req.Method, req.URL)
 }
 
-func (f *Forwarder) getDialer(ctx authContext, sess *clusterSession, req *http.Request) (httpstream.Dialer, error) {
+func (f *Forwarder) getSPDYDialer(ctx authContext, sess *clusterSession, req *http.Request) (httpstream.Dialer, error) {
```

Comment:
what is SPDY? could probably use a comment

Reply:
SPDY is an obsolete protocol that predeceased HTTP/2.
Unfortunately, Kubernetes uses it instead of HTTP/2, probably because it's far simpler than what HTTP/2 ends up being.